x-pack/plugins/infra/common/inventory_models/types.ts

```diff
@@ -285,7 +285,7 @@ export const ESTopHitsAggRT = rt.type({
   top_hits: rt.object,
 });
 
-interface SnapshotTermsWithAggregation {
+export interface SnapshotTermsWithAggregation {
```

Comment:
Because of TS4023 (Microsoft/TypeScript#5711) some additional interfaces need to be exported because we're emitting declarations:

```
x-pack/plugins/infra/server/routes/snapshot/lib/get_metrics_aggregations.ts:29:14 - error TS4023: Exported variable 'metricToAggregation' has or is using name 'SnapshotTermsWithAggregation' from external module "/Users/smith/Code/kibana/x-pack/plugins/infra/common/inventory_models/types" but cannot be named.
```